kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/GCSToBQWriter.java

```diff
@@ -103,9 +115,8 @@ public void writeRows(List<RowToInsert> rows,
 
     // Check if the table specified exists
     // This error shouldn't be thrown. All tables should be created by the connector at startup
-    if (bigQuery.getTable(tableId) == null) {
-      throw new ConnectException(
-          String.format("Table with TableId %s does not exist.", tableId.getTable()));
+    if (autoCreateTables && bigQuery.getTable(tableId) == null) {
```

Comment:
Suggest we do lazy table creation here too. Only do so when insert fail.

Reply:
It's not feasible to catch exception and retry during the process of writing rows from GCS to BQ since it uses built-in BigQuery Job definition thus cannot be modified. The `bigquery.getTable(tableId)` shouldn't be a concern here since this line of code has been existed for a while and never caused the throttle problem.

Reply:
Talked with @bingqinzhou offline, since integration test passes, we wouldn't worry too much.